### ISSUE TITLE
Review C36 form updates

### ISSUE THREAD

**Issue by aprocik1** [department-of-veterans-affairs/va.gov-team#80327]
## Description
We need to provide content recommendations for the updated Chapter 36 form.

## Tasks
- [x] KF: Provide recommendations
- [x] AP: Review
- [x] AP, KF: Find answers to outstanding questions
- [x] KF: Share with product team

## Criteria
- [x] The product team has our recommendations for the updated form

**Comment by aprocik1** [comment]
@katherine-fung, thanks for drafting content recommendations for the updated Chapter 36 form. I responded to all your comments in [the Figma file](https://www.figma.com/file/DEnVQXRrNvwJ3UkKkYYJ1Q/Ch-36-PCPG-2024-Updates?type=design&node-id=283-16287&mode=design&t=PSLe4KiavPI0rLoe-4).

Here are some outstanding questions I pulled out of our comments:
- Where should we communicate the transferred education benefits eligibility requirement for dependents?
- Do we need to reiterate the transferred education benefits eligibility requirement in the form flow?
- How do we typically talk about ending / separating from active duty?
- Is it possible for a service member to not be on active duty? (i.e. they're in the Reserves)
- Can a Veteran be on active duty? If so, does that make them a service member?

**Comment by katherine-fung** [comment]
@aprocik1, 


> - Where should we communicate the transferred education benefits eligibility requirement for dependents?
> 
> - Do we need to reiterate the transferred education benefits eligibility requirement in the form flow?
> 

I commented in the Figma that I had this wrong, and it's not actually transferred benefits that make a dependent eligible. It's that the dependent has to be eligible for Post-9/11 GI Bill or Montgomery GI Bill to qualify. I suggested alternate copy for the questions/copy in Figma comments. 

> How do we typically talk about ending / separating from active duty?

I asked in proj-va-content Slack

> Is it possible for a service member to not be on active duty? (i.e. they're in the Reserves)
> Can a Veteran be on active duty? If so, does that make them a service member?

I've asked Beth for guidance on this.

**Comment by aprocik1** [comment]
Thanks, @katherine-fung. I think the changes you made in the Figma file are great. Hopefully we're able to get answers to these remaining questions soon.

**Comment by katherine-fung** [comment]
@aprocik1, 
 
I dropped some new comments into the [Figma file](https://www.figma.com/file/DEnVQXRrNvwJ3UkKkYYJ1Q/Ch-36-PCPG-2024-Updates?type=design&node-id=110-4319&mode=design&t=Ac6J9jXAHqMudy7b-0) that may address this outstanding question. 

> How do we typically talk about ending / separating from active duty?

I've proposed changes based on the [design system's pattern for asking for service history](https://design.va.gov/patterns/ask-users-for/service-history). 

> Is it possible for a service member to not be on active duty? (i.e. they're in the Reserves)
> Can a Veteran be on active duty? If so, does that make them a service member?

I still don't have the answer to this question, but the changes I've proposed per the design system eliminate the need to specify "Veteran" or "service member" in all but 2 instances.
